hw1/PKCS-1.5.py

```diff
@@ -121,8 +123,10 @@ def enc_PKCS_1(self, d, ps=None):
             raise Exception("first byte must be nonzero 0 if bt=0")
 
         if ps is None:
-            ps = ?
-        eb = ?  # Encryption Block
+            # PS is at least `RSA_PKCS_1.min_pad_size-3` long, as
+            # it was validated that d is short enough.
+            ps = self.pad(self.k - 3 - len(d))
+        eb = b'\x00' + self.bt.to_bytes(1, byteorder='big') + ps + b'\x00' + d
```

Comment:
Add comment that if ps is passed it is not validated as it was implied from the requirements to only check the decryption and it is also usefull for testing to not check it.

Reply:
Done